src/features/onboarding/Welcome.js

```diff
@@ -36,38 +37,22 @@ const Welcome = () => {
             },
           })
         )
-        break
+        break;
+      }
 
-      case "newsletterOff":
-        dispatch(
-          updateUser({
-            email: userEmail,
-            profile: {
-              ...profile,
-              newsletter: "0",
-            },
-          })
-        )
-        break
+      case "addToHomeScreen": {
+        // Don't do some A2HS magic
```

Comment:
You need to do exactly the same as in the welcome case. Do not look at the old newsletter case as I tried to do something smart there that never really worked.